sdk/core/azure-core/tests/async_tests/test_pipeline_async.py

```diff
@@ -286,3 +287,25 @@ def send(*args):
     pos_retry = policies.index(retry_policy)
     assert pos_boo < pos_retry
     assert pos_foo > pos_retry
+
+    policies = [UserAgentPolicy(),
+                AsyncRetryPolicy(),
+                DistributedTracingPolicy()]
+    client = AsyncPipelineClient(base_url="test", policies=policies, per_call_policies=[boo_policy])
```

Comment:
Could we duplicate this test using all single instances for `per_call_policies`/`per_retry_policies`?